apis/common/v1/condition.go

```diff
@@ -169,26 +170,23 @@ func (s *ConditionedStatus) GetCondition(ct ConditionType) Condition {
 // ignoring the last transition time, to those already set.
 // Observed generation is updated if higher than the existing one.
 func (s *ConditionedStatus) SetConditions(c ...Condition) {
-	for _, new := range c {
+	for _, cond := range c {
```

Comment:
scope creep, but it is not good to use a semi-reserved word as a local var.

Reply:
We lint for this in c/c. Wonder if this repo's golangci-lint config is lagging behind.

Reply:
Made an issue to follow up.